### PR TITLE
[utils] Add test util for init model params to ones

### PR DESCRIPTION
**Pull request by RdoubleA**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #223
* #222
* #221
* __->__ #219
* #218

### CODE REVIEW

File: test/test_utils.py [hunk 1]
```diff
@@ -79,3 +80,8 @@ def assert_expected(actual: Any, expected: Any, rtol: float = None, atol: float
         atol=atol,
         msg=f"actual: {actual}, expected: {expected}",
     )
+
+
+def init_params_ones(model: nn.Module) -> None:
```
Comment (langong347) [inline test/test_utils.py:85]: Would it be too specific to init everything to 1? Or do you plan to make this configurable?

Comment (RdoubleA) [reply test/test_utils.py:85]: No plans at the moment, if we need to init to anything else we can make the change later or add a new function. Not sure what else we could init to for testing.

Comment (langong347) [reply test/test_utils.py:85]: Non-blocking Q:

The need to have deterministic model params is to avoid output shift due to model structure changes, right?

In general, I am not too sure about this approach (to instill artificial values). Statistically, this may skew certain properties. I suppose once our models stabilize, there will be less of the need to do so.

Just some food for thought: [TorchVision](https://github.com/pytorch/vision/blob/main/test/test_models.py#L602)'s model test design:
- set rng seed
- init the model
- pass in random inputs
- compare with expected outputs (stored as pickles, i.e., `test/expect/*.pkl`)
